main.cpp

```diff
@@ -1,22 +1,39 @@
 #include <windows.h>
 
-enum { id_button1 = 1, id_button2 };

```

Comment:
Zašto ste obrisali `enum`?
Ideja je bila da u kodu ne koristite (i ponavljate) konstante (`1`, `2`) za button id-eve.